### ISSUE TITLE
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.

### ISSUE THREAD

**Issue by alucard001** [lzybkr/PSReadLine#2924]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
I got the following crash report with the following info:


Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 25 Keys:
 UpArrow Ctrl+l Enter
 UpArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow LeftArrow Ctrl+l Enter
 Ctrl+l UpArrow Enter
 UpArrow Enter
 Ctrl+l

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -32.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
```
```


### Screenshot

![image](https://user-images.githubusercontent.com/1968244/137668833-7b9cfd03-9c15-4c22-a0da-fad9540a8ef1.png)


### Environment data

```console
PS Version: 5.1.19041.1237     
PS HostName: ConsoleHost       
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows   
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 181
BufferHeight: 46
```

### Steps to reproduce

Not very sure, it just pop up without any notice.

### Expected behavior

No expected behavior

### Actual behavior

Crash.

**Comment by alucard001** [comment]
You can freely close it if this bug did not mean anything.  Thank you.

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1315). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.
